tests/test_session.py

```diff
@@ -250,8 +250,7 @@ def test_execute_tui_commands(new_mesh_session, tmp_path=pyfluent.EXAMPLES_PATH)
     with open(file_path) as f:
         returned = f.readlines()
 
-    if os.path.exists(file_path):
```

Comment:
This step was redundant as the entire directory was being cleaned before starting the tests.